EIPS/eip-draft_bridged_tokens.md

```diff
@@ -0,0 +1,344 @@
+---
+title: Bridged Token Standard
```

Comment:
```suggestion
title: Bridged Token
```
The title field in the preamble: Should not include the word “standard” or any variation thereof;
https://eips.ethereum.org/EIPS/eip-1